.jenkins/github/pre-merge.jenkinsfile

```diff
@@ -7,43 +7,96 @@ VERSIONS = [
     changeList: "${env.CHANGE_ID}.${env.BUILD_NUMBER}.sdk" // Docker registry rejects versions with capital letters!
 ]
 LOCKS = [
-    sdk: "sdk-compile-${VERSIONS.changeList}"
+    build: "sdk-compile-${VERSIONS.changeList}",
+    agents: "sdk-agents-${VERSIONS.changeList}",
+    chef: "sdk-chef-${VERSIONS.changeList}"
 ]
 JOB_ABORTED = false
 ERROR_COUNT = 0
 
-node('jdk11') {
-  try {
-    job_options()
-    calculate_versions()
-    parallel(
-        'build and test': {
-          build_and_test()
-        },
-        'spin up instances': {
-          spinup_instances()
-        },
-        failFast: true
+podTemplate(containers: [
+    containerTemplate(
+        name: 'default',
+        image: 'registry.stage.c8y.io/ci/jdk:11',
+        ttyEnabled: true,
+        command: '/bin/sh -c',
+        args: "cat",
+        envVars: [
+            envVar(key: 'JENKINS_URL', value: 'http://jenkins.ci.svc.cluster.local:8080/'),
+            envVar(key: 'MAVEN_USER_HOME', value: '/home/jenkins/agent'),
+            envVar(key: 'DOCKER_HOST', value: 'tcp://localhost:2375')
+        ],
+        resourceRequestCpu: '3',
+        resourceRequestMemory: '8Gi',
+        resourceLimitCpu: '4',
+        resourceLimitMemory: '8Gi',
+        workingDir: '/home/jenkins/agent'
+    ),
+    containerTemplate(
+        name: 'dind-daemon',
+        image: 'registry.stage.c8y.io/ci/docker:19.03-dind',
+        command: 'dockerd-entrypoint.sh',
+        args: '',
```

Comment:
obsolete?

Reply:
but harmless... I'd have to check the base image if it does not define any...

Reply:
sure, don't bother